### ISSUE TITLE
java.lang.NoSuchMethodError: No static method applyCount

### ISSUE THREAD

**Issue by mtford90** [jhen0409/react-native-android-badge#3]
Hi there,

I'm getting this native exception when calling `setBadge`:

```
java.lang.NoSuchMethodError: No static method applyCount(Landroid/content/Context;I)Z in class Lme/leolin/shortcutbadger/ShortcutBadger; or its super classes (declaration of 'me.leolin.shortcutbadger.ShortcutBadger' appears in /data/app/com.bookingsapp-1/base.apk)
```

Any idea what I'm doing wrong here?

**Comment by yernandus** [comment]
Hello,
Having same error

**Comment by yernandus** [comment]
fixed, the + sign should not be used for gradle build, so I used the latest version which is 10

**Comment by jhen0409** [comment]
Fixed by #4. Thanks @yernandus.
